grpc-api/src/main/java/io/deephaven/grpc_api/util/GrpcUtil.java

```diff
@@ -57,7 +57,11 @@
         try (final SafeCloseable ignored = LivenessScopeStack.open()) {
             return lambda.call();
         } catch (final StatusRuntimeException err) {
-            log.error().append(err).endl();
+            if (err.getStatus().equals(Status.UNAUTHENTICATED)) {
+                log.debug().append("ignoring unauthenticated request: ").append(err).endl();
```

Comment:
thoughts on INFO so it shows up (and isnt just dropped entirely), but isn't visible in the UI? unless i'm mistaken, debug/trace logging are really only meant for "i'm debugging a problem" not "suppress this until i go looking for it", since it will just get dropped, won't be available for someone to look later.

Would be nice if we could junk the stack trace though, and just log where we're being called from...